data/map planets.txt

```diff
@@ -838,6 +839,7 @@ planet Darkrest
 	attributes hai station uninhabited
 	landscape land/station34
 	description `This station was mothballed sometime in the last sixty thousand years. The docking area has an atmosphere, and a status panel indicates that the station's generators are functional but shut down, with standby mechanisms running on solar power alone.`
+	description `	The original purpose of this station was to house Hai who had migrated from beyond the wormhole once it became clear that the Hai's empire was no longer tenable. However, today it is most notorious for an engineering malfunction that occurred when this station was still inhabited. In a freak accident, an entire habitation section of Darkrest lost power and depressurized, causing the deaths of countless civilians. Thousands of years later, no one knows for sure what caused the incident. Legends, stories, and conspiracy theories have blossomed from the event.`
```

Comment:
Not a fan of this one. Descriptions for these should be limited to what can be gleaned from observation.
Also, and event like this would be far more appropriate if there was a station in the same system as Darkwaste.

Reply:
I'm not sure how the Darkrest event would make more sense if the station was in the same system as Darkwaste.
I see your point about the issue with the description not really following with what you are able to observe. I'll try changing it.